.github/workflows/graphql.yml

```diff
@@ -13,15 +13,15 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
-    if: github.repository == 'meshery/meshery'
+    if: github.repository == 'meshery/meshery'32

```

Comment:
@amino19 this 32 sneaked in somehow :eyes:

Reply:
@DelusionalOptimist : My bad! :sweat_smile: